fiftyone/server/utils/datasets.py

```
@@ -5,30 +5,45 @@
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
-
+from typing import Any, Dict, List
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)